src/test/regress/expected/multi_router_planner.out

```diff
@@ -2041,6 +2041,61 @@ DEBUG:  Plan is router executable
 (6 rows)
 
 SET client_min_messages to 'NOTICE';
+-- test that a connection failure marks placements invalid
+SET citus.shard_replication_factor TO 2;
+CREATE TABLE failure_test (a int, b int);
+SELECT master_create_distributed_table('failure_test', 'a', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('failure_test', 4);
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+CREATE USER router_user;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+GRANT INSERT ON ALL TABLES IN SCHEMA public TO router_user;
```

Comment:
Unless I'm doing something wrong this works in the `enterprise-master` branch, it doesn't propagate by default maybe?